projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -0,0 +1,74 @@
+<ng-container *tuiLet="options.metadata | async as phoneMetadata">
+    <div
+        *ngIf="countriesNames$ | async as countriesNames"
+        tuiGroup
+        [tuiDropdown]="dropdown"
+        [tuiDropdownEnabled]="!readOnly"
+        [(tuiDropdownOpen)]="!computedDisabled && open"
+        (tuiActiveZoneChange)="onActiveZone($event)"
+    >
+        <tui-textfield
+            tuiChevron
+            class="t-country-select tui-group__auto-width-item"
```

Comment:
I would ideally drop those group classes someday in the future. Would be nice if we didn't use them here if possible, since we have local classes on both of those controls.

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/022347a56c0e1f04052fad24885b1a58e84b24d9